typescript/sdk/src/consts/environments/mainnet.json

```diff
@@ -17,7 +17,6 @@
     "multisigIsm": "0x9bDE63104EE030d9De419EEd6bA7D14b86D6fE3f",
     "create2Factory": "0xc97D8e6f57b0d64971453dDc6EB8483fec9d163a",
     "testRecipient": "0x36FdA966CfffF8a9Cdc814f546db0e6378bFef35",
-    "interchainAccountRouter": "0xE0Be420779cAd6E2bEA1E4F7C02F996D9ED1fCB5",
```

Comment:
Why remove?

Reply:
incompatible with the proxying

Reply:
but yeah if you want we can wait for the followup PR